modules/virt-creating-a-vm-from-a-default-os-image.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-a-vm-from-a-default-os-image_{context}"]
+= Creating a virtual machine from a default OS image
+
+After you have created a Persistent Volume Claim (PVC) for default OS images, you create a new virtual machine or virtual machine template from a default OS image that you uploaded to the PVC.
+
+[NOTE]
+====
+You can create virtual machines and virtual machine templates from default OS images with non-administrator user privileges. You need administrator privileges to upload and save default OS images to PVCs.
+====
+
+.Prerequisites
+
+* A PVC must exist that supports default OS images.
+* When you created the PVC, you selected the *Attach this disk to a VirtualMachine operating system* check box.
+
+.Procedure
+
+. In the {product-title} Web Console, click *Workloads > Virtualization* in the sidebar menu. The *Virtualization* page displays. Use onscreen Help or existing documentation to create virtual machine templates.
+
+. Create a virtual machine or virtual machine template from an uploaded default OS image.
+.. Select *Create Virtual Machine > New with Wizard*. The *Create Virtual Machine* wizard displays.
+.. On the *General* wizard page, select an operating system from the *Operating System* drop-down list that displays the *(Source available)* label next to the operating system and version name. The *(Source available)* label indicates default OS images are available for this operating system.
```

Comment:
Can lose the "drop-down" part:

```
*Operating System* list
```

Per ISG.

Reply:
Got it. I deleted "drop-down" in all content.